x-pack/test/fleet_api_integration/apis/agents_setup.ts

```diff
@@ -101,7 +101,7 @@ export default function (providerContext: FtrProviderContext) {
       );
     });
 
-    it('should create or update the fleet_enroll user if called multiple times with forceRecreate flag', async () => {
+    it.skip('should create or update the fleet_enroll user if called multiple times with forceRecreate flag', async () => {
```

Comment:
@elastic/fleet @nchaulet somehow this test is failing when I remove `fleet-agent-events`. The same test also fails on master if I enable v1 migrations for this test with the following:

```diff
--- a/x-pack/test/fleet_api_integration/config.ts
+++ b/x-pack/test/fleet_api_integration/config.ts
@@ -65,6 +65,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     kbnTestServer: {
       ...xPackAPITestsConfig.get('kbnTestServer'),
       serverArgs: [
+        '--migrations.enableV2=false',
         ...xPackAPITestsConfig.get('kbnTestServer.serverArgs'),
         ...(registryPort ? [`--xpack.fleet.registryUrl=http://localhost:${registryPort}`] : []),
       ],
```

We're targeting 7.12.1 for this fix, so please take a look and let me know if it's possible to fix this test or if we should skip it.

Reply:
I think it's okay to skip it, we are going to remove that API in 7.13.